tests/unit/ReadableStream.ts

```diff
@@ -155,6 +154,7 @@ registerSuite({
 			};
 		},
 
+		tests: {
```

Comment:
Do the lines in these `{}` need to be indented?

Reply:
Yes, this was simply for review purposes. I'll indent them before merging.